dbms/src/Common/ProfileEvents.cpp

```diff
@@ -114,6 +114,7 @@
     M(SelectedRanges, "Number of (non-adjacent) ranges in all data parts selected to read from a MergeTree table.") \
     M(SelectedMarks, "Number of marks (index granules) selected to read from a MergeTree table.") \
     \
+    M(CountOfMerges, "") \
```

Comment:
Need description.

Reply:
We don't have `Count` in the names of other metrics because counting is assumed.

Reply:
Let it be named `Merge`.